garcon/activity.py

```diff
@@ -253,7 +253,16 @@ def run(self):
         previous activity is consumed (context).
         """
 
-        activity_task = self.poll()
+        try:
+            activity_task = self.poll()
+        except Exception as error:
```

Comment:
Yes. I was hoping to have contextual example such as the one you mentioned here (aws rate limit). Sorry for the back and forth.